comp/swerve100/src/main/java/org/team100/frc2024/RobotContainer.java

```diff
@@ -187,6 +191,7 @@ public RobotContainer(TimedRobot robot) throws IOException {
                 poseEstimator,
                 swerveLocal,
                 driverControl::speed);
+        m_noteCamera = new CameraAngles(30, 67.5, 50, 832, 616, 0.71, notePositionDetector, m_drive);
```

Comment:
are you using the v2 camera or the v3 camera?  they have different FOV.  you could configure the camera type in the Camera class (which is like Identity for cameras) or you could pass the camera type through network tables, it's available in python as camera.camera_properties['Model'].